README.md

```diff
@@ -28,27 +28,22 @@ The included `example/` directory contains an example MirageOS unikernel which u
 
 ## Network Setup
 
-To get started, you'll need a "public" network (one from which the Internet is accessible) and a "private" network (one which doesn't have outside access; this will be provided by the unikernel once it's online).  Configure the unikernel with the correct public network information, and an IP address on the private network.  For example, to set up a unikernel with a public network on 192.168.3.1/24, and a private 10.0.0.0/24 network, if configured for Xen:
-
-```
-mirage configure -t xen --public-ipv4=192.168.3.1/24 --public-ipv4-gateway=192.168.3.254 --private-ipv4=10.0.0.1/24
-```
-
-Then follow the usual MirageOS workflow:
+To get started, you'll need a "public" network (one from which the Internet is accessible) and a "private" network (one which doesn't have outside access; this will be provided by the unikernel once it's online).  The public network, public gateway, and the private network will be defined at runtime, therefore you can follow the usual MirageOS workflow:
 
 ```
+mirage configure -t xen
 make depend
 make
 ```
 
-and start the unikernel as appropriate for the hypervisor:
+and start the unikernel as appropriate for the hypervisor, with the runtime arguments:
 
 ```
-sudo xl create simple_nat.xl -c
+sudo xl create simple_nat.xl -c --public-ipv4=192.168.3.1/24 --public-ipv4-gateway=192.168.3.254 --private-ipv4=10.0.0.1/24
```

Comment:
I'm not sure how you pass boot parameters with xl :/

Reply:
I suspect you need to use `cmdline=...` but I find it hard to find any examples using this...
```suggestion
sudo xl create simple_nat.xl -c 'cmdline=--public-ipv4=192.168.3.1/24 --public-ipv4-gateway=192.168.3.254 --private-ipv4=10.0.0.1/24'
```

Reply:
```suggestion
sudo xl create simple_nat.xl -c 'extra=--public-ipv4=192.168.3.1/24 --public-ipv4-gateway=192.168.3.254 --private-ipv4=10.0.0.1/24'
```

Reply:
I guess a workaround is to modify the example to use solo5-hvt instead of xen :D but it would be nice to know and get that mirage issue updated.

Reply:
As `mirage-bootvar-xen` is currently archived, I'll update the example to use one of the solo5 tenders :)
I'm not aware of users using mirage with Xen directly, and other targets have options: Qubes has `qvm-prefs --set mirage-xxx -- kernelopts '--public-ipv4=...'`, solo5-{hvt,spt} also have options, etc.

Reply:
`mirage-bootvar-xen` was consolidated into https://github.com/mirage/mirage-bootvar - a bit unfortunate the README was not updated with this information before archiving.

Reply:
oh, would youind to open an issue at mirage/mirage? I'll then briefly inarchive and update the README.